chi/network.py

```diff
@@ -539,7 +548,7 @@ def remove_all_routes_from_router(router_id):
         router_id (str): The router ID.
     """
     return remove_routes_from_router(router_id,
-        get_router(router_id)['routes'])
+        get_router(router_id).get('routes'))
```

Comment:
Under what circumstances is the response from `get_router` a dict, but it doesn't contain this key?

Reply:
I'm not sure, but when I tried dict[key] a KeyError was being thrown, so I switched to dict.get(key) instead. I'll experiment with it a little more and get back to you.

Reply:
On closer inspection, I wrote this change when the router dict was further embedded in get_router, so this threw a KeyError at times due to that. This has changed since then and I agree this modification is no longer relevant. This has been noted, and this deletion/addition have been reverted.